server/config/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = function(app, passport) {
   // fetched and seed our stores with data.
   // App is a function that requires store data and url to initialize and return the React-rendered html string
   app.get('*', function (req, res, next) {
-    App(req, res);
+    App.default(req, res);
```

Comment:
@kaizvn Are they likely to be changing it in babel 6? Perhaps the previous behaviour we got used to in ES5 was wrong. Haha.